src/opds_dumper.cpp

```diff
@@ -206,10 +240,26 @@ std::string OPDSDumper::categoriesOPDSFeed() const
   );
 }
 
+std::once_flag fillLanguagesFlag;
+void fillLanguagesMap() 
+{
+  for (auto icuLangPtr = icu::Locale::getISOLanguages(); *icuLangPtr != NULL; ++icuLangPtr) {
+    auto lang = *icuLangPtr;
+    const icu::Locale locale(lang);
+    icu::UnicodeString ustring;
+    locale.getDisplayLanguage(locale, ustring);
+    std::string displayLanguage;
+    ustring.toUTF8String(displayLanguage);
+    std::string iso3LangCode = locale.getISO3Language();
+    iso639_3.insert({iso3LangCode, displayLanguage});
+  }
+}
+
```

Comment:
Please move this into the unnamed namespace just under the definition of `iso639_3` so that

1. all code responsible for the languages map is in one place
2. global namespace is not polluted (which is important in a library)

Reply:
Done, thanks